tests/test_shared_id.py

```diff
@@ -0,0 +1,205 @@
+async def test_sharedId_in_different_navigable(websocket, context_id):
+    await goto_url(
+        websocket, context_id,
+        "data:text/html,<div some_attr_name='some_attr_value' "
+        ">some text<h2>some another text</h2></div>")
+
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": "document.querySelector('body > div');",
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": True
+            }
+        })
+
+    shared_id = result["result"]["value"]["sharedId"]
+
+    await goto_url(websocket, context_id, "data:text/html,some other page")
+
+    with pytest.raises(Exception) as exception_info:
+        await execute_command(
+            websocket, {
+                "method": "script.callFunction",
+                "params": {
+                    "functionDeclaration": "(arg)=>{return arg}",
+                    "this": {
+                        "type": "undefined"
+                    },
+                    "arguments": [{
+                        "sharedId": shared_id}],
+                    "awaitPromise": False,
+                    "target": {
+                        "context": context_id
+                    }
+                }
+            })
+
+    recursive_compare({
+        "error": "no such node",
+        "message": string_containing("different document")
+    }, exception_info.value.args[0])
+
+
+@pytest.mark.asyncio
+async def test_sharedId_not_found(websocket, context_id):
+    await goto_url(
+        websocket, context_id,
+        "data:text/html,<div some_attr_name='some_attr_value' "
+        ">some text<h2>some another text</h2></div>")
+
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": "document.querySelector('body > div');",
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": True
+            }
+        })
+
+    shared_id = result["result"]["value"]["sharedId"] + "9999"
+
+    with pytest.raises(Exception) as exception_info:
+        await execute_command(
+            websocket, {
+                "method": "script.callFunction",
+                "params": {
+                    "functionDeclaration": "(arg)=>{return arg}",
+                    "this": {
+                        "type": "undefined"
+                    },
+                    "arguments": [{
+                        "sharedId": shared_id}],
+                    "awaitPromise": False,
+                    "target": {
+                        "context": context_id
+                    }
+                }
+            })
+
+    recursive_compare({
+        "error": "no such node",
+        "message": string_containing("was not found")
+    }, exception_info.value.args[0])
+
+# TODO
```

Comment:
http://go/todo(bug)

if there's one, please add TODO(#123) to refer to the github issue tracked by this TODO

same for other TODOs

Reply:
Removed TODO, as it will be a part of the WPT part of the specification: https://github.com/GoogleChromeLabs/chromium-bidi/issues/238